src/Cake.Core/Scripting/ScriptProcessor.cs

```diff
@@ -85,10 +85,10 @@ public sealed class ScriptProcessor : IScriptProcessor
             installPath = installPath.MakeAbsolute(_environment);
 
             var result = new HashSet<FilePath>(PathComparer.Default);
-            if (analyzerResult.Addins.Count > 0)
+            if (addins.Any())
```

Comment:
`Possible multiple enumeration of IEnumerable`

Was thinking, either don't check if it contains any, pass `IReadOnlyCollection` or just let it be...